experiments/ClimaCore/sea-ice/README.md

```diff
@@ -0,0 +1,71 @@
+# **Simple Sea Ice Model**
+
+# Semtner's Zero-layer Model
+
+Sea ice is approximated as a 0-layer slab. This means that the model does not occupy a particular gridpoint between the mixed layer (ocean) and the atmosphere. It is essentially a column-wise ODE, with no horizontal transport between the columns. In the absence of ice, the sea-ice model reduces to the slab ocean formulation. The ice is assumed to have a negligible heat capacity, so there is no heat storage within it. The only store of energy comes from the ice thickness.
```

Comment:
This is a little unclear to me - "The ice is assumed to have a negligible heat capacity, so there is no heat storage within it. The only store of energy comes from the ice thickness."  I guess we mean that the temperature of the ice is fixed at the freezing point (c dT_ice /dt = 0 b/c c -> 0) but that amount of ice can change, which changes the energy stored within the ice.

Reply:
I guess this is just saying that the ice layer doesn't have a prognostic equation for T. I will rephrase.